web/cmd/main.go

```diff
@@ -1,40 +0,0 @@
-package main
-
-import (
-	"net/http"
-	"os"
-
-	"code.cloudfoundry.org/lager"
-	"github.com/concourse/concourse/web"
-	"github.com/concourse/concourse/web/proxyhandler"
-)
-
-func NewLogger() lager.Logger {
-	logger := lager.NewLogger("web")
-	logger.RegisterSink(lager.NewReconfigurableSink(lager.NewPrettySink(os.Stdout, lager.DEBUG), lager.DEBUG))
-	return logger
-}
-
-func main() {
-
-	logger := NewLogger()
-
-	proxyHandler, err := proxyhandler.NewHandler(logger, "http://localhost:8080")
-	if err != nil {
-		panic(err)
-	}
-
-	webHandler, err := web.NewHandler(logger)
-	if err != nil {
-		panic(err)
-	}
-
-	http.Handle("/api/", proxyHandler)
-	http.Handle("/auth/", proxyHandler)
-	http.Handle("/oauth/", proxyHandler)
-	http.Handle("/", webHandler)
-
-	if err = http.ListenAndServe(":8081", nil); err != nil {
-		logger.Error("server-error", err)
-	}
-}
```

Comment:
What was the purpose of this file? Was it possible to build the UI server by itself?

Reply:
Yup https://github.com/concourse/web/blob/8c95285e65efc043abcf6d285b011c01d309087b/cmd/main.go

Reply:
Yeah, I don't think we use it anymore now that the `docker-compose` flow is pretty smooth so I figured I'd just remove it.